tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -112,19 +112,21 @@
 		} else {
 			if ( ! pluginsList ) {
 				pluginsList = dofetch(
-					`${ host }/wp-json/jurassic.ninja/jetpack-beta/branches/${ repo }/${ currentBranch }`
+					`https://betadownload.jetpack.me/query-branch.php?repo=${ encodeURIComponent(
+						repo
+					) }&branch=${ encodeURIComponent( currentBranch ) }`
```

Comment:
I hate when prettier does this, it screws up syntax highlighting in vim. If you hate it too we could do like this
```suggestion
					// prettier-ignore
					`https://betadownload.jetpack.me/query-branch.php?repo=${ encodeURIComponent( repo ) }&branch=${ encodeURIComponent( currentBranch ) }`
```

Reply:
Yes, I was just looking for that syntax :)